xarray/tests/test_dataarray.py

```diff
@@ -410,6 +410,21 @@ def test_constructor_from_self_described(self):
         actual = DataArray(IndexVariable("foo", ["a", "b"]))
         assert_identical(expected, actual)
 
+    @requires_dask
+    def test_constructor_from_self_described_chunked(self):
+        import dask.array as da
+
+        data = da.from_array([[-0.1, 21], [0, 2]])
+        expected = DataArray(
+            data,
+            coords={"x": ["a", "b"], "y": [-1, -2]},
+            dims=["x", "y"],
+            name="foobar",
+            attrs={"bar": 2},
+        )
```

Comment:
```suggestion
        expected = DataArray(
            [[-0.1, 21], [0, 2]],
            coords={"x": ["a", "b"], "y": [-1, -2]},
            dims=["x", "y"],
            name="foobar",
            attrs={"bar": 2},
        ).chunk()
```
I think this should be the same.

Reply:
Indeed